asciidoc-confluence-publisher-client/pom.xml

```diff
@@ -47,6 +47,16 @@
             <artifactId>commons-lang</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
```

Comment:
We order dependencies by group-id and artifact-id, not scope (in order to avoid ordering changes if scope changes - see https://github.com/confluence-publisher/confluence-publisher/blob/master/CONTRIBUTING.md)